ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -46,16 +46,7 @@ LT_DPC:
   - LT
   - DPC
 
-# Noncanonical types
```

Comment:
@tasodorff is there a reason why this type was deleted?

Reply:
looks like it was moved to another file (LGRP)

Reply:
Correct, it was moved as it did no belong in this file.